R/convertClimateTrace.R

```diff
@@ -0,0 +1,36 @@
+#' Convert ClimateTrace data
+#'
+#' @md
+#' @param x A [`magpie`][magclass::magclass] object returned from
+#'          [`readClimateTrace()`].
+#'
+#' @return A [`magpie`][magclass::magclass] object.
+#'
+#' @author Pascal Weigmann
+#'
+#' @export
```

Comment:
I think export does not make sense for madrat functions, as they should never be called directly, but rather via readSource or calcOutput.

Reply:
Indeed, missed that after copying from another file!